thrift-gen/zipkincore/ttypes.go

```diff
@@ -741,7 +741,7 @@ func (p *BinaryAnnotation) String() string {
 // precise measurement decoupled from problems of clocks, such as skew or NTP
 // updates causing time to move backwards.
 //
-// For compatibilty with instrumentation that precede this field, collectors
+// For compatibiltiy with instrumentation that precede this field, collectors
```

Comment:
please do not make changes in the auto-generated files

Reply:
ah, my bad. Can you point me to the source so I can make the change there?